app/src/main/kotlin/co/netguru/baby/monitor/client/feature/server/ServerViewModel.kt

```diff
@@ -183,7 +184,7 @@ class ServerViewModel @Inject constructor(
         previewEnabled: Boolean = mutableCameraState.value?.previewEnabled == true,
         streamingEnabled: Boolean = mutableCameraState.value?.streamingEnabled == true
     ) {
-        mutableCameraState.postValue(CameraState(previewEnabled, streamingEnabled))
+        mutableCameraState.value = (CameraState(previewEnabled, streamingEnabled))
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/a946f01a2ff7f8917d0a20fd06bd6cd5cf3e6b3b/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/server/ServerViewModel.kt#L187"></span>Parentheses in (CameraState(previewEnabled, streamingEnabled)) are unnecessary and can be replaced with: CameraState(previewEnabled, streamingEnabled)</td>
    </tr>
  </tbody>
</table>